src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -190,7 +190,10 @@ where
 
         let cancel_state = match &schedule_status {
             ScheduleStatus::Ok => None,
-            ScheduleStatus::NoTask | ScheduleStatus::PickFailure => None,
+            ScheduleStatus::NoTask | ScheduleStatus::PickFailure => {
+                self.hummock_manager.report_scale_compactor_info().await;
+                None
+            }
```

Comment:
IIUC we only need to report scale info when a scale is possibly required. `NoTask` or `PickFailure` have no such indication. Perhaps only report when there's no idle compactor, or simply at a regular interval?

Reply:
yes, i prefer to regular interval